src/test/java/sk/freemap/gpxAnimator/ui/TestDurationSpinnerModel.java

```diff
@@ -15,7 +15,7 @@
     @BeforeAll
     public static void setup() {
         edgeCases.add(Long.MIN_VALUE);
-        edgeCases.add(Long.MIN_VALUE);
+        edgeCases.add(Long.MAX_VALUE);
```

Comment:
Please revert this change. I have already fixed it and refactored the test to make use of `@ParameterizedTest`. I renamed the file to `DurationSpinnerModelTest`, too, which now results in a merge conflict if you alter the old file.

Tip: Before starting work on a project, it is always a good idea to do a `git pull upstream master` first. This will update your local repository with all changes from the original upstream repository.

Reply:
If you don't know, how to revert a change which is a part of a commit which is a part of a pull request, don't worry, I can do this manually while merging.